src/fidesops/service/connectors/query_config.py

```diff
@@ -392,9 +425,10 @@ def get_formatted_query_string(
         """Returns a query string with double quotation mark formatting as required by Snowflake syntax."""
         return f'SELECT {field_list} FROM "{self.node.node.collection.name}" WHERE {" OR ".join(clauses)}'
 
-    def format_key_map_for_update_stmt(self, key_map: Dict[str, Any]) -> List[str]:
+    def format_key_map_for_update_stmt(self, fields: List[str]) -> List[str]:
         """Adds the appropriate formatting for update statements in this datastore."""
-        return [f'"{k}" = :{k}' for k in key_map]
+        fields.sort()
```

Comment:
What's the significance of this sorting in the flow?

Reply:
I'm not sure, my guess is that it was added for debugging/printing that was done in earlier drafts.  

https://github.com/ethyca/fidesops/commit/e96f14a5e85ecf6023e48997304c651e7e6e20be

It shouldn't have a practical significance other than making logs easier to read.